cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -678,7 +678,7 @@
                         "webhook": {
                             "body": "{ \"ok\": \"true\" }",
                             "request": "GET /?cmd=success HTTP/1.1\r\nHost: 127.0.0.1:49999\r\nUser-Agent: goflow-testing\r\nAccept-Encoding: gzip\r\n\r\n",
-                            "response": "HTTP/1.1 200 OK\r\nContent-Length: 16\r\nContent-Type: text/plain; charset=utf-8\r\nDate: Wed, 11 Apr 2018 18:24:30 GMT\r\n\r\n{ \"ok\": \"true\" }",
+                            "response": "HTTP/1.1 200 OK\r\nContent-Length: 16\r\nContent-Type: text/plain; charset=utf-8\r\nDate: Wed, 11 Apr 2018 18:24:30 GMT\r\n\r\n",
```

Comment:
since body is included separately above - seems preferable to not duplicate it here

Reply:
Agree we don't want it twice.. but this seems awkward as request and response don't really pair well now. Maybe we should only have response and parse it for body when it is asked for?